bindata/manifests/switchdev-config/files/switchdev-configuration-before-nm.sh.yaml

```diff
@@ -13,6 +13,8 @@ contents:
       exit
     fi
 
+    which jq 2>&1 >/dev/null || (echo "Required tool jq is not found in PATH." && exit 1)
```

Comment:
nit: stderr redirection should go after the stdout one:

```
bash-5.2$ which fancy-command 2>&1 >/dev/null
which: no fancy-command in (/usr/local/bin:/usr/bin:/bin:/usr/local/sbin:/usr/sbin:/sbin:/var/lib/snapd/snap/bin:...)

bash-5.2$ which fancy-command  >/dev/null 2>&1
bash-5.2$ 
```

we can even get rid of all the redirections and the command after `||`, as `which` exits when jq is not found with a good error message.

Reply:
Thanks, adjusted.